lib/rock/webapp/tasks/api.rb

```diff
@@ -250,29 +250,49 @@ def get_operation(name_service, name, operation_name)
                         end
                     end
 
-                    desc 'connect a port, /connect&to=taskname&port=portname, optional &type=buffer&size=10'
+                    desc 'connect a port, /connect&to=hostname/taskname&port=portname&host=hostname, optional &type=buffer&size=10'
```

Comment:
Don't allow for hostname both in the to= and the host=, pick one. Moreover, the "wording" is not hostname but name_service, stick to it.